srv/modules/runners/push.py

```diff
@@ -72,6 +72,14 @@ def proposal(filename = "/srv/pillar/ceph/proposals/policy.cfg", dryrun = False)
     pillar_data.output(common)
     return True
 
+def __create_dirs(path, root):
+    try:
+        os.makedirs(path)
+    except OSError as err:
+        if err.errno == errno.EACCES:
+            print "ERROR: Cannot create dir {}".format(path)
```

Comment:
does the print statement also get displayed when raising the exception ? (or do we also need a log.error? )